src/index.tsx

```diff
@@ -17,7 +17,7 @@
 import React, { useState, useEffect, useRef, useCallback } from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import { BrowserRouter as Router, Redirect, Route, useLocation } from 'react-router-dom'
+import { HashRouter as Router, Redirect, Route, useLocation } from 'react-router-dom'
```

Comment:
what's the difference?

Reply:
this works better with Github pages: https://create-react-app.dev/docs/deployment/#notes-on-client-side-routing
the url will be like `https://devnet.alephium.org/#/blocks` with a hash character in between